web/styles/popovers.css

```diff
@@ -1190,4 +1218,120 @@ ul {
         width: 16px;
         height: 16px;
     }
+
+    .zulip-icon {
+        position: relative;
+        top: -1px;
+    }
+}
+
+#gear-menu-dropdown {
+    padding: 5px 0;
+    box-shadow:
```

Comment:
Is it worth making this box-shadow a variable as well? Even if only to make this area of the code a bit more readable?

Reply:
Yeah, done.